specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-10-01-preview/Clusters.json

```diff
@@ -0,0 +1,567 @@
+          "$ref": "#/definitions/Identity",
+          "description": "The identity of the resource."
+        },
+        "sku": {
+          "$ref": "#/definitions/ClusterSku",
+          "description": "The sku properties."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ClusterProperties",
+          "description": "Log Analytics cluster properties."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "description": "The top level Log Analytics cluster resource container."
+    },
+    "ClusterListResult": {
+      "properties": {
+        "nextLink": {
+          "description": "The link used to get the next page of recommendations.",
+          "type": "string"
+        },
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Cluster"
+          },
+          "description": "A list of Log Analytics clusters."
+        }
+      },
+      "description": "The list clusters operation response."
+    },
+    "keyVaultProperties": {
+      "description": "The key vault properties.",
+      "properties": {
+        "keyVaultUri": {
+          "description": "The Key Vault uri which holds they key associated with the Log Analytics cluster.",
+          "type": "string"
+        },
+        "keyName": {
+          "description": "The name of the key associated with the Log Analytics cluster.",
+          "type": "string"
+        },
+        "keyVersion": {
+          "description": "The version of the key associated with the Log Analytics cluster.",
+          "type": "string"
+        }
+      }
+    },
+    "ClusterSku": {
+      "description": "The cluster sku definition.",
+      "properties": {
+        "capacity": {
+          "description": "The capacity value",
+          "type": "integer",
+          "format": "int64"
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the SKU.",
+          "enum": [
+            "CapacityReservation"
+          ],
+          "x-ms-enum": {
+            "name": "ClusterSkuNameEnum",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "Identity": {
+      "properties": {
+        "principalId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The principal ID of resource identity."
+        },
+        "tenantId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The tenant ID of resource."
+        },
+        "type": {
+          "type": "string",
+          "description": "The type of identity used for the resource. The type 'SystemAssigned, UserAssigned' includes both an implicitly created identity and a set of user assigned identities.",
+          "enum": [
+            "SystemAssigned",
```

Comment:
You should support "None" to allow users to remove the identity and support a combination of user and systemassigned "SystemAssigned,UserAssigned"

Reply:
We do not support None, user cannot remove an identity, only to switch to other one.

Reply:
You'll either need to support it or show explicit approval from the MSI team.  See armwiki.azurewebsites.net/authorization/managed_identities/MSIOnboardingSteps.html#7-new-add-support-for-identity-type-none

Reply:
Ok, i'm addressing it now. thank you.